packages/calling-stateful-client/review/calling-stateful-client.api.md

```diff
@@ -151,6 +157,7 @@ export interface StatefulCallClient extends CallClient {
     createView(callId: string | undefined, participantId: CommunicationIdentifierKind | undefined, stream: LocalVideoStreamState | RemoteVideoStreamState, options?: CreateViewOptions): Promise<void>;
     disposeView(callId: string | undefined, participantId: CommunicationIdentifierKind | undefined, stream: LocalVideoStreamState | RemoteVideoStreamState): void;
     getState(): CallClientState;
+    modifyState(modifier: CallStateModifier): void;
```

Comment:
One thing unrelated to the PR for discussion: Since we have this one, do you think it is still necessary to add clearState()?

Or we can ask people to use this modifier to clear state

Reply:
`clearState` isn't needed with this. We can just provide a modifier to clear the state.